config/webpack/loaders/devSiteSrcCode.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
Instead of creating a new loader, add the 'SourceCodeBadge' to the package loader:
https://github.com/cerner/terra-dev-site/blob/master/config/webpack/loaders/devSitePackage.js